hedgehog/hedgehog.cabal

```diff
@@ -134,3 +134,18 @@ test-suite test
     , semigroups                      >= 0.16       && < 0.19
     , text                            >= 1.1        && < 1.3
     , transformers                    >= 0.3        && < 0.6
+
+executable dieharder-input
```

Comment:
I've added this as executable because, as a test-suite, IIRC, Travis CI tries to run it and eventually it'll timeout as it takes ~25 minutes to finish.

This way, we can run the DieHarder tests on demand, as executable, via

```
dieharder-input | dieharder -a -g 200
```

(About those `-a -g 200` command line arguments, see [DieHarder manpage](http://manpages.ubuntu.com/manpages/xenial/man1/dieharder.1.html#contenttoc6).)